mobile/src/main.jsx

```diff
@@ -51,12 +51,20 @@ const renderAppWithPersistedState = persistedState => {
     }
   }
 
+  function pingOnceADayWithState () {
```

Comment:
👀 it's the morning ^^ I did not see it's a refactoring of 'resume' & 'deviceready' event.